### PR TITLE
[closes #90] new crop: Asparagus

### LINKED ISSUE #90: new crop Asparagus
add a new crop, asparagus! asparagus is a slow growing crop, but it sells for a high value. seeds are also not that cheap.

### PR DESCRIPTION
**Pull request by lstebner**
new crop: Asparagus. takes 9 days to grow, and unlocks at level 24.

## growth stages

| name | days |
|-------|-----|
| seed | 4 |
| growing | 5 |
| total | 9 |

## some images:

slightly incorrect images as the SPROUT stage was removed after these screenshots and pulled out into a separate feature request (#107)

Screen Shot 2021-06-05 at 11 06 22 AM
Screen Shot 2021-06-05 at 11 18 28 AM
Screen Shot 2021-06-05 at 11 09 31 AM
Screen Shot 2021-06-05 at 11 09 49 AM

### CODE REVIEW

**Comment by jeremyckahn** [comment]
I love it! I will make some for a proper review in the coming days once I have some time. Great work so far! 🙂

File: src/enums.js
Comment (lstebner) [inline src/enums.js:84]: i think eslint did this.. is that expected though?

File: src/data/items.js
Comment (jeremyckahn) [inline src/data/items.js:117]: Let's make this tier 4 (and move it below the other crops in this file). Similarly to what I mentioned in [this comment](https://github.com/jeremyckahn/farmhand/issues/104#issuecomment-855300761), I'd like each tier to only have three crops.

**Review by jeremyckahn — CHANGES_REQUESTED** [review]
This PR is off to a great start! I really like the art design for the Asparagus. I did some manual regression testing and it seems that this is breaking every non-Asparagus crop's growth cycle because of the new `SPROUT` stage. The test failures are legitimate and need to be fixed before this can be merged.

To give a little more detail, Farmhand's implementation is currently designed around the assumption that there are only `SEED`, `GROWING`, and `GROWN` crop stages, so we'll need to account for some crops having a `SPROUT` stage and some not. The easiest solution would be descope this PR and only have the `SEED`, `GROWING`, and `GROWN` Asparagus states, but I'll leave it up to you @lstebner to decide how you'd like to proceed with it.

**Comment by lstebner** [comment]
thanks for the detailed explanation @jeremyckahn . sounds like that's a later issue indeed. i see no reason to hold up asparagus for a new growth stage. I added that mostly out of curiosity. we can discuss if we would like to add a new growth stage separately. i will clean this PR up

File: src/data/items.js
Comment (lstebner) [reply src/data/items.js]: updated

**Comment by lstebner** [comment]
all set for review @jeremyckahn . i left the images because i don't think it hurts anything, but can remove if you prefer.

File: src/data/levels.js
Comment (jeremyckahn) [inline src/data/levels.js:58]: 💯

File: src/img/index.js
Comment (jeremyckahn) [inline src/img/index.js:18]: Let's remove this to keep it out of the app bundle.

File: src/img/index.js
Comment (jeremyckahn) [inline src/img/index.js:98]: This line should be removed as well.

File: src/img/items/asparagus-sprout.piskel
Comment (jeremyckahn) [inline src/img/items/asparagus-sprout.piskel:1]: Let's definitely keep this in for future use! 🙂

**Review by jeremyckahn — CHANGES_REQUESTED** [review]
This is looking and working great! There's just a few small cleanup items I'd like to get addressed and this will be ready to merge. Thanks for sticking with this!

File: src/img/index.js
Comment (lstebner) [reply src/img/index.js]: good eye! removed the sprout refs from this file

**Review by jeremyckahn — APPROVED** [review]
LGTM! Thanks for adding a new crop and iterating on this PR. Feel free to merge when you're ready and run a `patch` release to ship it to players.

Nice work! 🎉

**Pull request merged**
